### PR TITLE
Fixing metric relationship types by resolving to inputs prior to edge finding.

### PR DESCRIPTION
**Pull request by mmcdermott**
This is a bit hacky, and we should move this code to depend on expressions eventually.

Also, this also needs to be put into master before next deploy, but I'm hoping we can just merge this into the epic then the epic into master and sort it all in one swoop.

### CODE REVIEW

File: src/lib/engine/space.js [hunk 1]
```diff
@@ -45,20 +45,19 @@ export function toDSpace(spaceId, graph, organizationFacts) {
...
     let edges = {}
     edges.inputs = dSpace.edges.filter(i => i.output === s.id).map(e => e.input)
     edges.outputs = dSpace.edges.filter(i => i.input === s.id).map(e => e.output)
     edges.inputMetrics = edges.inputs.map(i => dSpace.metrics.find(m => m.id === i))
-    return {
-      ...s,
-      edges,
-      guesstimate: withInputFn(s.guesstimate),
-    }
+    return { ...s, edges }
```
Comment (OAGr) [inline src/lib/engine/space.js]: why did you remove ``guesstimate`` here?  Just checking to make sure that wasn't a mistake.

Comment (mmcdermott) [inline src/lib/engine/space.js]: I moved it above. Right now, we resolve edges based on the guesstimate's input and readable IDs, so the expressions need to be resolved into inputs before we compute the edges.
